views/components/Config/ConfigCardAvatar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Avatar from '@material-ui/core/Avatar'
+
+import { colors } from '../../../constants'
+
+const ConfigCardAvatar = ({ ownsConfig, configOwner, userAvatar }) => {
+  if (ownsConfig) return userAvatar
+  if (!ownsConfig) {
```

Comment:
based on the condition above this will always be true, right? Do we ever get to the `else` here?

Reply:
I don't think we're talking about the same thing. On line 7 we're explicitly returning `userAvatar` if `ownsConfig` is `true`. Therefore, we shouldn't need to check if `ownsConfig` is `false` here.